umpleonline/ump/manualexamples/E074DuplicateParallelStateMachineNameInParallelRegion1.ump

```diff
@@ -0,0 +1,28 @@
+// The following example generates the error
```

Comment:
Why there should be an issue with `t2`? They are completely in different states. The issue with names happens when a composite state have two nested state machine with the same names.